pkg/sdk/procedures_ext.go

```diff
@@ -108,6 +118,75 @@ func procedureDetailsFromRows(rows []ProcedureDetail) (*ProcedureDetails, error)
 		v.NormalizedArguments = args
 	}
 
+	if v.ExternalAccessIntegrations != nil {
+		if p, err := parseFunctionOrProcedureExternalAccessIntegrations(*v.ExternalAccessIntegrations); err != nil {
+			errs = append(errs, err)
+		} else {
+			v.NormalizedExternalAccessIntegrations = p
+		}
+	} else {
+		v.NormalizedExternalAccessIntegrations = []AccountObjectIdentifier{}
+	}
+
+	if v.Secrets != nil {
+		if p, err := parseFunctionOrProcedureSecrets(*v.Secrets); err != nil {
+			errs = append(errs, err)
+		} else {
+			v.NormalizedSecrets = p
+		}
+	} else {
+		v.NormalizedSecrets = map[string]SchemaObjectIdentifier{}
+	}
+
+	if v.Packages != nil {
+		if p, err := parseFunctionOrProcedurePackages(*v.Packages); err != nil {
+			errs = append(errs, err)
+		} else {
+			// TODO [SNOW-1850370]: merge these and unit test
+			switch strings.ToUpper(v.Language) {
+			case "JAVA", "SCALA":
```

Comment:
Nit: use these as enums values. Also, add a comm explaining what we're doing here, it's not clear.

Reply:
+1